gradle/libs.versions.toml

```diff
@@ -151,6 +153,9 @@ minikdc = { group = "org.apache.hadoop", name = "hadoop-minikdc", version.ref =
 immutables-value = { module = "org.immutables:value", version.ref = "immutables-value" }
 commons-cli = { group = "commons-cli", name = "commons-cli", version.ref = "commons-cli" }
 sun-activation = { group = "com.sun.activation", name = "javax.activation", version.ref = "sun-activation-version" }
+kafka-clients = { group = "org.apache.kafka", name = "kafka-clients", version.ref = "kafka" }
+kafka_2_12 = { group = "org.apache.kafka", name = "kafka_2.12", version.ref = "kafka" }
```

Comment:
Using the name `kafka` is better ,  use the `kafka_2_12` may be make users mistakenly believe that using the version of kafka 2.12

Reply:
done